File: Java/TaskSix.java
import java.util.Scanner;
public class TaskSix{

    public static void main (String [] args){

    Scanner input = new Scanner (System.in);

    int total = 0;
   
 
    System.out.println("Enter a number: ");
    int number = input.nextInt();
       for (int count = 1; count <= number; count++){
    total += count;
       
}
     System.out.println(total);
}
    


}

File: JavaScript/Gate Two/TaskSix.js
const prompt = require("prompt-sync")();

let number = parseInt(prompt("Enter a number: ")):

let total = 0 


for(let count = 1; count < number + 1; count++){

    total = count + count

    result = '$(count) + $(count) = $(total)'

console.log(result);
    }
